sale_exception/tests/test_sale_exception.py

```diff
@@ -133,7 +133,7 @@ def test_sale_order_exception(self):
         # Simulation the opening of the wizard sale_exception_confirm and
         # set ignore_exception to True
         so_except_confirm = self.sale_exception_confirm.with_context(
-            **{"active_id": so1.id, "active_ids": [so1.id], "active_model": so1._name}
+            **{"id": so1.id, "ids": [so1.id], "active_model": so1._name}
```

Comment:
This causes your tests to fail. Please revert.

Reply:
And you can avoid such notation 
```suggestion
            with_context(active_id=so1.id, active_ids=so1.ids, active_model=so1._name)
```